examples/catalog/lib/expansion_tile_sample.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:math';
```

Comment:
I would revert the sample changes and make it a separate PR, since I'm not sure if this is what the maintainers of the catalog would want

Reply:
I wasn't planning to leave it, it was just for testing, like I mentioned here: `I can commit these changes so that you can test too if you prefer and remove them before the merge.`

Reply:
I've removed the changes to avoid confusion as they are available [here](https://gist.github.com/arthurdenner/1780d3594a06eb06b94474d45c23301a).

Reply:
Sorry about that, when I read your initial comment I thought you were referencing your implementation of the header and icon tweens and not the sample!